src/views/FAQ.vue

```diff
@@ -32,33 +32,29 @@
           <a
             href="https://aeternity.com/"
             target="_blank"
-          >aeternity blockchain</a>.
-          <br>Anyone, anywhere may use the platform.
-          <br>Other websites may use the platform.
-          <br>All have uncensored, equal ability to be Superhero.
-          <br>As P2P – the only ones who are not part of Superhero intermediaries.
-          <br>No 3
-          <sup>rd</sup> party custodians will control
-          content, or your ability to tip or receive tips.
-          <br>Superhero is you and you and you and you… Without any “them” in-between.
+          >aeternity blockchain</a>. <br>Anyone,
+          anywhere may use the platform. <br>Other websites may use the platform. <br>All have
+          uncensored, equal ability to be Superhero. <br>As P2P – the only ones who are not part
+          of Superhero intermediaries. <br>No 3 <sup>rd</sup> party custodians will control
+          content, or your ability to tip or receive tips. <br>Superhero is you and you and you
+          and you… Without any “them” in-between.
```

Comment:
the linter requires `<br>` to be the last on the line with a break after it (AFAIK)
```suggestion
          >aeternity blockchain</a>.<br>
          Anyone, anywhere may use the platform.<br>
          Other websites may use the platform.<br>
          All have uncensored, equal ability to be Superhero.<br>
          As P2P – the only ones who are not part of Superhero intermediaries.<br>
          No 3 <sup>rd</sup> party custodians will control content,
          or your ability to tip or receive tips.<br>
          Superhero is you and you and you and you… Without any “them” in-between.
```